src/list.builder.js

```diff
@@ -23,23 +23,33 @@ var EasyAutocomplete = (function (scope) {
 			return listBuilder;
 		};
 
-		this.updateCategories = function (listBuilder, data) {
+    this.updateCategories = function(listBuilder, data) {
+      if (configuration.get("categoriesAssigned")) {
 
-			if (configuration.get('categoriesAssigned')) {
+        listBuilder = [];
 
-				listBuilder = [];
+        if (configuration.getOptions('categories') !== undefined) {
```

Comment:
Using configuration.get("categories") that will give you the default value if we do not set categories 
![Screen Shot 2021-05-25 at 2 16 51 PM](https://user-images.githubusercontent.com/2033729/119491988-40b7ca80-bd67-11eb-8246-601b809e8385.png)
that why add a new function the return the options that sent to check if the categories or not.